app/controllers/ontologies_controller.rb

```diff
@@ -209,130 +208,126 @@ def content_serializer
 
     render 'ontologies/content_serializer', layout: nil
   end
-
+  
+  def redirect_ontology(accpet_header)
```

Comment:
put this in the ontologies_redirection_controller.rb and create a new route for this, e.g `/ontologies/ACRONYM/download?format=FORMAT`